amp/TEMPLATE/reamp/modules/activity/preview/utils/ApiHelper.jsx

```diff
@@ -1,14 +1,16 @@
 import {fetchJson, postJson} from 'amp/tools/index';
+import translate from '../utils/translate';
 
 export default class ApiHelper {
     static extractErrors(errors, obj) {
         const errorMessages = [];
         if (errors) {
             errors = Array.isArray(errors) ? errors : [errors];
             errors.forEach((error) => {
-                for (const key in error) {
-                    const messageKey = 'amp.preview:server-errors-' + key;
-                    const message = {messageKey};
+                for (const _key in error) {
+                    const message = {
+                        messageKey: translate("activityNotFound"),
```

Comment:
this code shall remain generic. see how  ErrorHelper.showErrors work and where you have to add the key to initial translations